build/checkstyle/tachyon_checks.xml

```diff
@@ -47,6 +47,11 @@
     <property name="fileExtensions" value=".java"/>
   </module>
   <module name="RegexpSingleline">
+    <property name="format" value="TODO[^\(]"/>
```

Comment:
Does this help?

http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/TodoCommentCheck.html